src/hooks/useApproveAndDeposit.ts

```diff
@@ -113,7 +113,10 @@ export function useApproveAndDeposit(
         token: BasicToken | undefined,
       ): Promise<void> => {
         if (!token || !library) {
-          enqueueToast("error", "Token or library is not loaded")
+          enqueueToast(
+            "error",
+            "There was a problem loading the token or library",
+          )
           throw new Error("Token or library is not loaded")
```

Comment:
I think throwing here will result in an uncaught error - we can just console.error instead. I think generally we should have ~2 types of error messages: one for the user eg "There was a problem with your transaction" displayed as a UI notification, and one for debugging or technical users eg "library is undefined" + stack trace, logged to the console and Sentry.